api/proto/teleport/legacy/types/types.proto

```diff
@@ -3371,7 +3379,7 @@ message AccessCapabilitiesRequest {
   bool SuggestedReviewers = 3 [(gogoproto.jsontag) = "suggested_reviewers,omitempty"];
   // ResourceIDs is the list of the ResourceIDs of the resources we would like to view
   // the necessary roles for.
-  repeated ResourceID ResourceIDs = 4 [
+  repeated types.ResourceID ResourceIDs = 4 [
```

Comment:
There's no need to use the protobuf package name if we're in the same package, even if the message is defined in a different file.

```suggestion
  repeated ResourceID ResourceIDs = 4 [
```